pkg/ext/stores/tokens/tokens.go

```diff
@@ -1520,3 +1576,19 @@ func ttlGreater(a, b int64) bool {
 
 	return a > b
 }
+
+// typecheck store type against the various interfaces we (have to) implement
```

Comment:
This is implied (idiomatic Go).
```suggestion
```

Reply:
done. commit 6434fb02ab93bbf500610c0d6214ca9daab4515e